iohub/reader.py

```diff
@@ -262,11 +262,21 @@ def print_info(path: StrOrBytesPath, verbose=False):
                 print("Zarr hierarchy:")
                 reader.print_tree()
                 positions = list(reader.positions())
+                total_GB_uncompressed = (
+                    len(positions) * (positions[0][1][0].nbytes) / 1e9
```

Comment:
THis would be confusing if showing 0.0 GB for <100 MB. Maybe try to mimic the behavior of `du -h`?

Reply:
e.g. https://web.archive.org/web/20111010015624/http://blogmag.net/blog/read/38/Print_human_readable_file_size

Reply:
Zarr also does this: https://zarr.readthedocs.io/en/v2.18.3/_autoapi/zarr.core.Array.html#zarr.core.Array.info

```
import zarr
z = zarr.zeros(1000000, chunks=100000, dtype='i4')
z.info
Type               : zarr.core.Array
Data type          : int32
Shape              : (1000000,)
Chunk shape        : (100000,)
Order              : C
Read-only          : False
Compressor         : Blosc(cname='lz4', clevel=5, shuffle=SHUFFLE, blocksize=0)
Store type         : zarr.storage.KVStore
No. bytes          : 4000000 (3.8M)
No. bytes stored   : 320
Storage ratio      : 12500.0
Chunks initialized : 0/10
```